Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -1189,6 +1189,11 @@ public int getThemeId()
 
   private static boolean isVRInstalled(Context context)
   {
+    if (context.getPackageName().compareTo(EmulationActivity.VR_PACKAGE) == 0)
+    {
+      return false;
+    }
```

Comment:
This doesn't match what the method name implies that this method does. You should either put this outside of this method or change the method name.

Also, `compareTo` can be replaced with `equals` here too.